runtime/pack/dist/opt/netrw/autoload/netrw/fs.vim

```diff
@@ -0,0 +1,167 @@
+" FUNCTIONS IN THIS FILE ARE MEANT TO BE USED BY NETRW.VIM AND NETRW.VIM ONLY.
+" THESE FUNCTIONS DON'T COMMIT TO ANY BACKWARDS COMPATIBILITY. SO CHANGES AND
+" BREAKAGES IF USED OUTSIDE OF NETRW.VIM ARE EXPECTED.
+
+let s:slash = &shellslash ? '/' : '\'
```

Comment:
Will this only be sourced on Microsoft Windows where &shellslash exists?

Reply:
Shellslash should be present either in Windows or UNIX the only thing that should be changing is the value of it

Reply:
The value of `&shellslash` is always 0 on Unix.

Reply:
`:help shellslash` still says

```
	{only for MS-Windows}
	...
	'shellslash' only works when a backslash can be used as a path
	separator.  To test if this is so use: >
		if exists('+shellslash')
```

and  `exists('+shellslash')` is a common idiom to [check for a Microsoft Windows shell](https://github.com/rayhan-wijaya/my-vim-fugitive/commit/652c27a46bb3d20a0909bde7f690a37a7515f3bf). It's safer to check for its existence

Reply:
Okay, this might have changed. All these existence checks must be around for a reason

Reply:
I believe is done as a legacy thing for version of vim that do not support this feature. Since netrw supports for now only the latest vim version this is not an issue